instrumentation/vertx/vertx-kafka-client-3.6/javaagent/build.gradle.kts

```diff
@@ -34,8 +34,8 @@ testing {
 
         // the "library" configuration is not recognized by the test suite plugin
         if (latestDepTest) {
-          implementation("io.vertx:vertx-kafka-client:+")
-          implementation("io.vertx:vertx-codegen:+")
+          implementation("io.vertx:vertx-kafka-client:latest.release")
+          implementation("io.vertx:vertx-codegen:latest.release")
```

Comment:
was it picking up SNAPSHOT version? could you tell from where?

Reply:
no it was picking up `5.0.0.CR1`, `latest.release` is also used by the usual latest dep magic and works with https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/29da88e7cf6b8898805c0c8ddc8095477d475ce6/conventions/src/main/kotlin/io.opentelemetry.instrumentation.base.gradle.kts#L34